spec/features/course/experience_points_record_management_spec.rb

```diff
@@ -7,10 +7,17 @@
   with_tenant(:instance) do
     let(:course) { create(:course) }
     let(:course_student) { create(:course_student, :approved, course: course) }
-    let!(:record) { create(:course_experience_points_record, course_user: course_student) }
-    let!(:inactive_record) do
+    let(:manual_record) { create(:course_experience_points_record, course_user: course_student) }
+    let(:inactive_record) do
       create(:course_experience_points_record, :inactive, course_user: course_student)
     end
+    let(:record) do
+      points_record = create(:course_experience_points_record, course_user: course_student)
+      points_record.actable = points_record
```

Comment:
If I stub it like `allow(stub).to receive(:manually_awarded?).and_return(false)`, `@experience_points_record` will not return false in the controller and the view will render incorrectly. Perhaps I could do the same thing (i.e. let a record act as itself), but shift the logic to factory girl?